### PR TITLE
Logs page

### PR DESCRIPTION
**Pull request by RishiVaya**
image added logs page. Logs shown as a table with creation and update times and "show logs" that opens a dialog with log message and an option to download logs as txt file

### CODE REVIEW

File: src/leo-client-app/src/components/logModal.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
+import * as React from 'react';
```
Comment (quinnha) [inline src/leo-client-app/src/components/logModal.tsx:1]: import only what you need not everything

File: src/leo-client-app/src/components/logModal.tsx [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+  const url = URL.createObjectURL(file);
+  const link = document.createElement("a");
+  link.download = "logFile.txt";
+  link.href = url;
+  link.click();
+  handleClose();
+}
+
+  return (
+    <React.Fragment>
```
Comment (quinnha) [inline src/leo-client-app/src/components/logModal.tsx:29]: why a react fragment and not a div/stack/whatever?

**Review by quinnha — CHANGES_REQUESTED** [review]
why is your pr description empty? add description of changes, a video of showing what was changed as well

File: src/leo-client-app/src/components/logModal.tsx (hunk 1 above)
Comment (RishiVaya) [reply src/leo-client-app/src/components/logModal.tsx]: fixed

File: src/leo-client-app/src/components/logModal.tsx (hunk 2 above)
Comment (RishiVaya) [reply src/leo-client-app/src/components/logModal.tsx]: just a dialog now

Comment (quinnha) [reply src/leo-client-app/src/components/logModal.tsx]: u didnt answer the question lmfao

File: src/leo-client-app/src/components/logModal.tsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+        onClose={handleClose}
+        aria-labelledby="alert-dialog-title"
+        aria-describedby="alert-dialog-description"
+      >
+        <DialogTitle id="alert-dialog-title">
+          {"Satellite Logs dialog"}
+        </DialogTitle>
+        <DialogContent>
+          <DialogContentText id="alert-dialog-description">
+            {JSON.stringify(logData?.data?.message)}
```
Comment (Umang-Rajkarnikar) [inline src/leo-client-app/src/components/logModal.tsx:42]: Just realized that message will already be a string so we don't need to stringify it

File: src/leo-client-app/src/components/logs.tsx [hunk 4]
```diff
@@ -0,0 +1,113 @@
...
+  Typography,
+} from "@mui/material";
+import React, { useEffect, useState } from "react";
+import LogDialog from "./logModal";
+
+const Logs: React.FC = () => {
+
+  // TODO: Dynamicall get satelliteId from somewhere
+  const satelliteId = "655acd63d122507055d3d2ea";
+  const [logs, setLogs] = useState<any>([]);
```
Comment (Umang-Rajkarnikar) [inline src/leo-client-app/src/components/logs.tsx:21]: Just a q, are we doing this for poc

File: src/leo-client-app/src/pages/satellite-logs/index.tsx [hunk 5]
```diff
@@ -0,0 +1,21 @@
+"use client";
```
Comment (Umang-Rajkarnikar) [inline src/leo-client-app/src/pages/satellite-logs/index.tsx:1]: what does this mean

Comment (quinnha) [reply src/leo-client-app/src/pages/satellite-logs/index.tsx:1]: rends client side so you can use hooks iirc

File: src/leo-client-app/src/components/logModal.tsx (hunk 3 above)
Comment (RishiVaya) [reply src/leo-client-app/src/components/logModal.tsx]: removed!

File: src/leo-client-app/src/components/logModal.tsx (hunk 2 above)
Comment (RishiVaya) [reply src/leo-client-app/src/components/logModal.tsx]: I am experimenting stuff. If I get smthing to work, I can optimize it after

**Review by Umang-Rajkarnikar — APPROVED** [review]
lgtm

**Pull request merged**